packages/lodestar/src/network/util.ts

```diff
@@ -61,8 +61,10 @@ export function isRequestSingleChunk(method: Method): boolean {
   return Methods[method].responseType === MethodResponseType.SingleResponse;
 }
 
-export function eth2ResponseTimer<T>(): (source: AsyncIterable<T>) => AsyncGenerator<T> {
+export function eth2ResponseTimer<T>(stream: Stream): (source: AsyncIterable<T>) => AsyncGenerator<T> {
   const controller = new AbortController();
+  // @ts-ignore
```

Comment:
why does it complain here?

Reply:
it can't get latest version of `libp2p-ts` although I do `yarn cache clean`

Reply:
does it work on ci? It would be better to investigate this error instead of doing ignore like this. Maybe you have it linked or something?

Reply:
I guess if I point to a different github branch then point back to master it should work, will give it a try